### PR TITLE
Rwchan116/create event page

### PR DESCRIPTION
**Pull request by rwchan116**
- event page with basic styling
- Create event icon and navigate to /event/create
- Route to /event/edit/${eventId} and /view/${eventId} based on event role (organizer vs attendee)
- Common header component

### CODE REVIEW

**Comment by rwchan116** [comment]
@samjeyam @vijayenthran @carolSequeira @jermm - Please review

File: fancy-team-server/server.ts [hunk 1]
```diff
@@ -1,6 +1,6 @@
 'use strict';
-import express from 'express';
-import cors from 'cors';
+import * as express from 'express';
+import * as cors from 'cors';
```
Comment (samjeyam) [inline fancy-team-server/server.ts]: please sync against master  - npm i and then try tsc to see if we could avoid this change

Comment (rwchan116) [reply fancy-team-server/server.ts]: Works with what's in master, so I have removed my change

**Comment by rwchan116** [comment]
@vijayenthran - Please review 

CC @samjeyam @jermm @carolSequeira

**Comment by rwchan116** [comment]
Merging it now.

**Pull request merged**